projects/observability/src/shared/components/cartesian/cartesian-chart.component.test.ts

```diff
@@ -185,13 +185,23 @@ describe('Cartesian Chart component', () => {
           name: 'test series 1',
           color: 'blue',
           type: CartesianSeriesVisualizationType.Column,
+          assignedGroup: {
+            id: 1,
+            groupName: 'first',
+            specName: 'sum(foo)'
```

Comment:
why does it need spec name? That should be unrelated to this api. Group Name should be sufficient, isn't it?

Reply:
+1 - specs should not be in the chart code, that's a data concept.

Reply:
Fixed, used name instead! some code changes that as well, please check